src/sections/selectNode.tsx

```diff
@@ -20,47 +21,66 @@ function Section1() {
     (store: Store) => store.auth.nodes
   ).map((elem: any, index: number) => {
     return {
-      name: elem.apiEndpoint,
+      name: elem.localName ?  `${elem.localName} (${elem.apiEndpoint})` : elem.apiEndpoint,
+      localName: elem.localName,
       value: index,
       apiEndpoint: elem.apiEndpoint,
       apiToken: elem.apiToken,
     };
   });
-  const connecting = useAppSelector(
-    (store: Store) => store.auth.status.connecting
-  );
-  const connected = useAppSelector(
-    (store: Store) => store.auth.status.connected
-  );
+  const connecting = useAppSelector((store: Store) => store.auth.status.connecting);
+  const connected = useAppSelector((store: Store) => store.auth.status.connected);
+  const loginData = useAppSelector((store: Store) => store.auth.loginData);
+  
+  const [searchParams, setSearchParams] = useSearchParams();
```

Comment:
should `setSearchParams` be `set_searchParams`? Or we only do that for states

Reply:
Ok, please update it